spec/controllers/provider_foreman_controller_spec.rb

```diff
@@ -504,8 +503,8 @@
       allow_any_instance_of(User).to receive(:get_filters).and_return(user_filters)
       Classification.seed
       quota_2gb_tag = Classification.where("description" => "2GB").first
-      Classification.bulk_reassignment(:model      => "ConfiguredSystem",
-                                       :object_ids => @configured_system.id,
+      Classification.bulk_reassignment(:model      => "ManageIQ::Providers::Foreman::ConfigurationManager",
+                                       :object_ids => @config_mgr.id,
```

Comment:
@bdunne Did this one go backwards?

Reply:
@h-kataria helped to get this test passing.  
> it's due to the fact that we removed match_via_descendants from tree building code it wasnt getting the correctly tagged object back